discord/guild.py

```diff
@@ -1591,6 +1591,72 @@ async def create_role(self, *, reason=None, **fields):
         # TODO: add to cache
         return role
 
+    async def edit_role_positions(self, *, positions, reason=None):
```

Comment:
```suggestion
    async def edit_role_positions(self, positions, *, reason=None):
```

I feel making this keyword-only is too strict.